thamos/data/defaultThoth.yaml

```diff
@@ -1,30 +1,47 @@
----
 #
-# This default Thoth configuration file placed in a root of a repo. Please adjust values listed below.
+# This is  Thoth's configuration file placed in a root of a repo used by Thamos
+# CLI as well as by Thoth bots. Please adjust values listed below as desired.
 #
-
-# TODO: Adjust Thoth's host.
-host: thoth-user-api.redhat.com
+host: thoth-user-api.redhat.com  # TODO: Adjust Thoth's host.
 tls_verify: true
-# One of testing, stable, latest.
+
+# Recommendation type - one of testing, stable, latest:
 recommendation_type: stable
-runtime_environment:
+
+# Format of requirements file, currently supported is only Pipenv:
+requirements_format: pipenv
+
+# Operating system for which the recommendations should be created:
+operating_system:
+  name: fedora
+  version: '29'
+
+# Hardware information for the recommendation engine:
+hardware_information:
   cpu_family: 6
   cpu_model: 94
+
+# Software configuration of runtime environment:
+runtime_environment:
+  python_version: '3.6'
+  cuda_version: null
```

Comment:
having it set to 'null' is just to give an example? its not required?

Reply:
It will not be required, just to state this config option exist.